app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -1,8 +1,11 @@
 <%
-  add_gem_component_stylesheet("layout-for-public")
+  blue_bar_background_colours = ["browse"].freeze
 
+  add_gem_component_stylesheet("layout-for-public")
+  blue_bar_background_colour ||= blue_bar_background_colours.include?(blue_bar_background_colour) ? blue_bar_background_colour : nil
```

Comment:
I don't understand what's going on here... sorry

Reply:
This line checks to see if the background colour specified is in the list of blue_bar_background_colours. If it isn't then it sets blue_bar_background_colour to nil otherwise it sets it to the background colour specified.

Reply:
OK I think the naming is _confusing_ so can we rename them pls? One as allowed colours maybe? The other as specified colours? Trying to spot one letter difference isn't user friendly

Reply:
Ah actually since it's a constant, the naming convention is that it should be in all caps. Have changed that.